src/en/proposals/fairlysadpanda-revs-uplink.md

```diff
@@ -0,0 +1,159 @@
+
+To do this, the Head Revolutionaries need to compromise the central security framework that controls and monitors the space station. This is done by obtaining, by any means, one or more Department Head IDs, breaking into the bridge, and hacking into the CentComm Uplink Terminal. After a timer, shortened each time a new Head ID is inserted into the machine, the Revolutionaries gain complete control over the station and win the round.
+
+If the Head Revolutionaries are neutralized before this happens, CentComm will send a dismissable emergency shuttle to pick up the crew.
+
+## Background
+
+The current Revolutionaries game mode ("Revs") as implemented is a problematic game mode that has lead to:
+
+- Repeated revisions to anti-metagaming rules, via both SOP and Space Law
+- A large number of complaints and discussions about improvements on Discord including, but not limited to:
+  - Revs causing a murderhobo tide that Security has to fight with murderhobo violence
+  - Boring stalemates and unclear round state, especially for the loyalist crew
+  - The endless feels-bad forced round-removal of discovered Head Revs and Heads Of Staff, sometimes for over an hour
+  - Paint-by-numbers round activities entirely focussed on flashes and MindShield implants.
+- Drain on Admins, both due to players demanding ERT revents in response to a discovered revolution and rules enforcement.
+
+Revs is badly in need of a refactor to encourage a more dynamic round that encourages paranoia from Security and smart play from the Revolutionaries. This document forms one half of the "Revs 2.0" refactor, alongside a to-be-written replacement of the Flash/MindShield conversion game mechanics. For the original overview document, [see here](https://github.com/space-wizards/docs/pull/132).
+
+# TL;DR
+
+The current "kill all Heads" win condition for Revolutionaries is replaced.
+
+The bridge Communications Computer is upgraded to a Centcomm Security Uplink terminal.
+
+This terminal is pretty hard to destroy, internally-powered, bolted to the floor and very heavy. If the terminal somehow gets destroyed (a Singuloose, immovable rod, etc) then CentComm will detect this and dispatch the needed machine board to make another to the trade station.
+
+This Computer works like a Communications Computer, but represents the station's connection to CentComm. This connection can be switched off by inserting one of a set of keycards: one of the original Department Head IDs (but not the Captain's spare). This starts a timer. Each additional Head ID reduces this timer. The timer can be disabled by accessing the terminal and cancelling it.
+
+If this timer expires, and it's a Revs round, the Revs win. Even if it isn't, all cards on the station automatically become All Access.
+
```

Comment:
What's to stop heads just destroying, walling in, or otherwise removing the terminal every round as there's no systems preventing this (plus we now have to consider specific gamerules for maps which to my knowledge no other gamerule has to do or will).

Reply:
Yup, this is the big question with a "capture the point" objective and it's been a struggle since captains were throwing the nuke disks out of airlocks on Goon.

If this is impossible to solve then it means modes like this simply cannot work, which would be a bit concerning as it limits SS round design space significantly.

To be honest though I'm stumped on how to solve this further. It's really hard to solve powergaming/metagaming.

Reply:
I think to prevent walling, maybe you could partially invert the idea and introduce a "dead man's switch" with this console. If nobody interacts with the console using the Head IDs on a regular and partially-randomized timer basis, they could lose control of elements of the station via powernet lockdowns and restrictions on cargo, etc. That at least guarantees that the console must be accessible, at least intermittently if nothing else. It continues to incentivize the seizing of the ID cards, as the only way to stop this altogether is to use the IDs in the console and disengage from CentCom's authority (which is an automatic loss condition for the heads if they do this themselves, naturally, and would probably incur ERT/Deathsquad absent a revolution). It also adds an IC reason for people to believe in the revolution - if the heads keep failing to satisfy their job requirements and the station repeatedly enters lockdowns and restrictions from CentCom, the crew will get progressively more mutinous, with or without the gamemode in play. 
Possibly even failing the lockdown timer several times in a row would automatically generate revolutionaries.

Reply:
Some ideas so far include centralizing some Command functions into the console or building in some of the comms functionality. I do want the Terminal to do something good and cool though, a "oh, cool" moment, not a "oh..." moment.

This is another thing I'm going to try and workshop with designers on the Discord.